yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -217,6 +215,20 @@
 </ul>
 </li>
 </ul>
+<t>The _COMPAT modifier string is "sticky". Once a revision of a module 
+has a modifier in the revision label, then all descendants of that 
+revision will also have a modifier. The modifier can change from 
+_compatible to _non_compatible in a descendant revision, but the 
+modifier would never change from _non_compatible to _compatible 
+or be removed. The persistency of the _non_compatible modifier 
+ensures that comparisions of revision labels doesn't give the false 
+impression of compatibility between two potentially non-compatible 
+revisions. If _non_compatible was removed, for example between revisions 
+3.3.2_non_compatible and 3.3.3 (where 3.3.3 was simply an editorial 
+change), then comparing revision labels of 3.3.3 back to an ancestor 
+3.0.0 would look like they are backwards compatible when they are 
+not (since 3.3.2_non_compatible was in the chain of ancestors and 
+introduced an NBC change).</t>
```

Comment:
I know we informally use NBC, but I thought we pulled that out of the draft in favor of spelling it out.

Reply:
OK. Will update to non-backwards-compatible (which we seem to use regularly in module versioning and YANG semver drafts).